src/util/json_expr.cpp

```diff
@@ -16,7 +16,7 @@ Author: Peter Schrammel
 #include "config.h"
 
 #include "json_expr.h"
-
+#include <java_bytecode/expr2java.h>
```

Comment:
That will require that any application built from CBMC source code links against java_bytecode. That doesn't seem to be desirable.

Looking at the changes, however, I don't see why you'd need it?